### PR TITLE
Fix build with Qt 5.11_beta3 (dropping qt5_use_modules)

### LINKED ISSUE #4: A change in Qt 5.11_beta3 leads to errors
Arch GNU/Linux, `bionic` branch

```
CMake Error at src/CMakeLists.txt:108 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete 

Same as https://github.com/ubports/ubuntu-download-manager/issues/4

### PR DESCRIPTION
**Pull request by vanyasem**
Closes #4 

Tested on Arch GNU/Linux

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -105,7 +105,7 @@ if (TRUST_STORE_MIR_AGENT_ENABLED)
     core/trust/mir/prompt_main.cpp
   )
 
-  qt5_use_modules(trust-prompt Core Gui Qml Quick)
+  target_link_libraries(trust-prompt Qt5::Core Qt5::Gui Qt5::Qml Qt5::Quick)
```
Comment (mardy) [inline src/CMakeLists.txt]: Move these modules inside the same block defined a couple of lines below.

Comment (vanyasem) [reply src/CMakeLists.txt]: Should I redo the PRs that were already merged that have the same issue then?

Comment (mardy) [reply src/CMakeLists.txt]: No, let those be, it's a minor cosmetic issue (I hope) :-)

Comment (vanyasem) [reply src/CMakeLists.txt]: Well, that is a cosmetic issue in fact, as I tested all of the PRs before sending them :P

**Comment by vanyasem** [comment]
All good now

**Review by mardy — APPROVED** [review]
LGTM, thanks! :-)

**Pull request merged**